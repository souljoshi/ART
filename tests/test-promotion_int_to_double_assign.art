// Test integer promotion in assinging int to double
// FAILING IN CODEGEN

int main()
{
	double a = 5;
	double b = 10.5;
	printf(a);
	printc('\n');
	printf(b);
	printc('\n');
	a = 9.5;
	b = 4;
	printf(a);
	printc('\n');
	printf(b);
	printc('\n');
}
