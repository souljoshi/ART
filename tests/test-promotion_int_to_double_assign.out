5.000000
10.500000
9.500000
4.000000

